src/v2/view-builder/internals/BaseFooter.js

```diff
@@ -27,8 +27,22 @@ export default View.extend({
    */
   links: [],
 
+  /**
+   * View
+   * adds any view to the footer in footer info section
+   */ 
+  footerInfo: null,
+
+  /**
+   * Boolean
+   * If true then 'Back to sign in' does not get added
+   */
+  noBackToSignInLink: false,
```

Comment:
negations are harder to reason about, `hasBackToSignInLink` is easier to read

Reply:
How is the "Back to Sign In" link related to the new Sign Up footer?

Reply:
Not related. some views needs to hide `Back to sign in` link and this is how they can do it.. for eg Signin

> > > For v2 widget, if profile enrollment is enabled, the Sign Up link moves to this new footer section? If profile enrollment is not enabled, does the Sign Up link in the new footer show up?
> > 
> > 
> > @sherizada-okta ahh.. did you mean when profile enrollment is enabled but denied ? let me check that.
> 
> @pradeepdewda-okta I mean by default, a profile enrollment policy is not setup, so there should be no "Sign Up" link?

By default the profile enrollment is enabled i think.. but it's always denied. Yeah so in that case i have to hide this section all together..

Reply:
> negations are harder to reason about, hasBackToSignInLink is easier to read

Idea is by default we add `Back to sign in` link to every view.. If a view doesn't want it then they can use this parameter.